For profiling, apply this patch and invoke `bundle install`.
profiling is enabled for every requests and profiling data will be
stored in 'tmp/profile'.

--- a/Gemfile
+++ b/Gemfile
@@ -55,3 +55,4 @@ gem 'bootstrap-sass'
 gem 'bootstrap_form'
 gem 'mini_magick'
 gem 'puma'
+gem 'ruby-prof'
--- a/config.ru
+++ b/config.ru
@@ -2,3 +2,4 @@
 
 require ::File.expand_path('../config/environment', __FILE__)
 run Rails.application
+use Rack::RubyProf, path: 'tmp/profile'
